### ISSUE TITLE
G3 - "Search" heading 2 appears above heading 1

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#133]
The local search has a hidden heading 2 that occurs before the heading 1 page title. The h2 should appear underneath the h1 or should be removed.

e.g. https://aoda.web.uoguelph.ca/oac/college-boreal-collaboration

**Comment by mmafe** [comment]
Applied as part of hotfix_20150807.3.0.

**Comment by jszasz** [comment]
Confirmed.
